subsys/bluetooth/controller/hal/nrf5/radio.c

```diff
@@ -68,16 +68,28 @@ void radio_phy_set(u8_t phy, u8_t flags)
 	case BIT(0):
 	default:
 		mode = RADIO_MODE_MODE_Ble_1Mbit;
+
+#if defined(CONFIG_SOC_NRF52840)
+		*(volatile u32_t *)0x4000173c &= ~0x80000000;
```

Comment:
I understand this is a workaround, but can you confirm that the source code where you took it from does not include a comment about its purpose? Same applies to all the similar expressions below